drivers/led/lp5569.c

```diff
@@ -159,6 +180,8 @@ static const struct led_driver_api lp5569_led_api = {
 #define LP5569_DEFINE(id)						\
 	static const struct lp5569_config lp5569_config_##id = {	\
 		.bus = I2C_DT_SPEC_INST_GET(id),			\
+		.enable_gpio = GPIO_DT_SPEC_GET_OR(DT_DRV_INST(id),	\
```

Comment:
There is `GPIO_DT_SPEC_INST_GET_OR` which can be used without `DT_DRV_INST`.

`GPIO_DT_SPEC_INST_GET_OR(id, enable_gpios, {0})`

Reply:
Got it